src/library/browse/browsetablemodel.cpp

```diff
@@ -471,3 +473,10 @@ QAbstractItemDelegate* BrowseTableModel::delegateForColumn(const int i, QObject*
     }
     return nullptr;
 }
+
+void BrowseTableModel::stopBrowseThread() {
```

Comment:
The name of this function is inappropriate as the comment tells. It doesn't stop the browse thread, it just drops one of multiple references.

-> `resetBrowseThread()` or `releaseBrowseThread()`?